third_party/llm_engine/llm_engine.bzl

```diff
@@ -36,9 +40,38 @@ def llm_engine():
 
 def _impl(repository_ctx):
     http_proxy = repository_ctx.os.environ.get("http_proxy", "")
+    if http_proxy == "":
+        http_proxy = repository_ctx.os.environ.get("HTTP_PROXY", "")
+    
     https_proxy = repository_ctx.os.environ.get("https_proxy", "")
+    if https_proxy == "":
+        https_proxy = repository_ctx.os.environ.get("HTTPS_PROXY", "")
+    
     OpenVINO_DIR = repository_ctx.os.environ.get("OpenVINO_DIR", "")
-    result = repository_ctx.execute(["cat","/etc/os-release"],quiet=False)
+
+    if _is_windows(repository_ctx):
+        OpenVINO_DIR = OpenVINO_DIR.replace("/", "\\\\").replace("\\", "\\\\")
+        out_lib_dir = '"runtime/lib/Release"'
+        lib_name = '"openvino_genai.lib"'
+        out_libs = 'out_static_libs = ["openvino_genai.lib"]'
```

Comment:
please try similarly to drogon integration
out_static_libs = [
            "drogon.lib",
            "trantor.lib"
        ]

it should be possible without replying on strings